tests/integration/modules/test_ssh.py

```diff
@@ -128,6 +128,9 @@ def test_get_known_host_entries(self):
             )
 
     @pytest.mark.slow_test
+    @pytest.mark.skipif(
+        salt.utils.platform.is_photonos() is True, reason="Skip on PhotonOS"
```

Comment:
**question** can we say *why* we need to skip on photon here?

Reply:
(same for the rest of these)

Reply:
A lot of these will likely be removed once we figure out the SSH key thing.